<launch>
    <rosparam file="$(find mimi_common_pkg)/config/object_mapping.yaml" command="load" ns="object_mapping" />
    <rosparam file="$(find mimi_common_pkg)/config/action_state.yaml" command="load" ns="action_state" />
    <rosparam file="$(find mimi_common_pkg)/config/demo.yaml" command="load" ns="/location_dict" />

    <node pkg="mimi_common_pkg" name="enter_the_room" type="enter_the_room.py"/>
    <node pkg="mimi_common_pkg" name="localize_object" type="localize_object.py"/>
    <node pkg="mimi_common_pkg" name="exe_action_plan" type="exe_action_plan.py"/>
    <include file="$(find mimi_common_pkg)/launch/approach_person.launch"/>
</launch>
